enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -137,6 +147,42 @@ public void backupCanNotBePerformedOverBackupProtocol() throws Exception
                         "--name=defaultport" ) );
     }
 
+    @Test
+    public void dataIsInAUsableStateAfterBackup() throws Exception
+    {
+        // given database exists
+        Cluster cluster = startCluster( recordFormat );
+
+        // and the database has indexes
+        createIndexes( cluster.getDbWithAnyRole( Role.LEADER ).database() );
+
+        // and the database is being populated
+        Thread thread = new Thread( () -> repeatedlyPopulateDatabase( clusterLeader( cluster ).database() ) ); // populate db with number properties etc.
+        oneOffShutdownTasks.add( thread::stop ); // kill thread
```

Comment:
please **DO NOT** use thread.stop, if you wonder why please read javadoc of Thread.stop